src/web/lib/components/App/index.js

```diff
@@ -99,7 +102,7 @@ export const AppComponent = props => {
 
   const customImages = (theme.images.custom_backgrounds || []).map(item => {
     const customImage = { ...item };
-    customImage.image = themeCustomImages[item.name].image;
+    customImage.image = themeCustomImages[item.name] && themeCustomImages[item.name].image;
```

Comment:
so this happens when the user saves a theme with a background image and then the user deletes the image - via the X button here - and does not delete the associated theme. We could delete the associated theme/s but my understanding was we wanted to leave these unless the user explicitly X’d these from the “Saved themes” tab. But this does cause some issues  - like `image.name` being void. Also, the theme will not be exported with the image that is deleted. I don’t exactly remember why we decided this was okay. I think it's because are future enhancements coming around this so for now this was okay. I believe with the util function I set it up, this shouldn't be an issue now since we are not including images that have been removed. I thought about updating the actually theme that references this but this was a quicker solution to address this.

With that being said, one thing I just noticed is if you select a theme in “Saved theme,” and go back to the “Custom backgrounds” tab, it only shows the images that are in localStorage images that are associated with the theme that was just selected in the “Saved theme” tab. Initially, I thought the idea of the “Custom background” tab was to show all the background images that have been uploaded but now seeing this I am not sure. I didn't realize you could click the associated theme and remove the image that way.  This is why I am removing the code I had added above (when the user goes into selecting a theme workflow, this doesn’t work as expected anyway). So given that and that you can access and potentially remove images via the “Saved themes”, I am hoping this is sufficient for now. If we do decide we want custom backgrounds to show all images that are in localstorage, I am hoping this can done in a follow up issue since it’s more involved than I initially thought.